### ISSUE TITLE
default should not be a reserved word in an object literal

### ISSUE THREAD

**Issue by jamestalmage** [eslint/eslint#3762]
See discussion on #3759

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by sindresorhus** [comment]
Relevant: https://mathiasbynens.be/notes/javascript-properties

**Comment by nzakas** [comment]
This issue didn't have enough information. Please update your description with the specific request.

**Comment by jamestalmage** [comment]
```js
{
default: 'foo'
}
```
should be allowed, but `eslint` complains it is a reserved word and needs to be in quotes, but that is wrong. Reserved works as property names are allowed in object literals, there is no need to quote them.

**Comment by nzakas** [comment]
@jamestalmage It sounds like you're reporting a bug, and if so, please see the first comment on this issue for what the required information is.

**Comment by ilyavolodin** [comment]
Closing as there was no response. Feel free to reopen with additional information.

**Comment by michaelficarra** [comment]
I suspect @jamestalmage was writing a block statement with a label named `default`, thinking it was an object initialiser.

**Comment by jamestalmage** [comment]
@michaelficarra you suspect wrong

The links provided by myself and @sindresorhus answer all 4 requirements posted by your bot and then some. However, since that appears insufficient I have made a repro here: https://github.com/jamestalmage/eslint-3762-repro

**Comment by jamestalmage** [comment]
So the issue seems to be the `keywords: true` argument in the config.

Does not appear this is an `eslint` issue at all. I've recently switched from `jshint`/`jscs` to `xo`, so I've not really delved into config specifics.
